docs/walkthrough/create-training-data.md

```diff
@@ -92,6 +92,16 @@ During fine-tuning, we're optimizing two models in parallel.
 At the model saving time, you will discover, we are saving two models to your local directory. 
 ```
 
+### Query-Document Scores
+
+To prepare data for query document pairs together with their similarity scores,
+each row must contain three columns includes two sentences and a similarity score.
+
+```markdown
+The weather is nice, The weather is beautiful, 0.9
+The weather is nice, The weather is bad, 0
```

Comment:
these are both comments on the weather, so I would assume that they are still somewhat similar, right? Maybe I am misunderstanding how similarity is defined but I would assume that dissimilar sentences would be more like this
```suggestion
The weather is nice, The weather is beautiful, 0.9
The weather is nice, I need to walk my dog, 0
```

Reply:
I did indeed misunderstand, you can ignore these suggestions :+1: